src/app/proposals/new/newProposalCards.data.tsx

```diff
@@ -36,10 +36,14 @@ export const newProposalCards: NewProposalCardData[] = [
     cardTitle: 'Grant',
     textBlock: (
       <>
-        <p>Community vote to allocate treasury funds for Grants</p>
         <p>
-          What&apos;s in it for the user, short explanation&hellip; Lorem ipsum dolor sit amet, consectetur
-          adipisicing elit. Amet, velit itaque consequatur cupiditate aliquid neque.
+          Community backed - your proposal is voted on inside the DAO, aligning funding with ecosystem
+          priorities.
+        </p>
+        <p>
```

Comment:
Does this paragraph have the correct font-family?”

Reply:
QA passed this as OK - in any case the scope of the ticket is not to edit the paragraph itself, rather than the text